nix/sources.nix

```diff
@@ -0,0 +1,174 @@
+# This file has been generated by Niv.
```

Comment:
This is a generated file that you can completely ignore.

Reply:
this is also generated by the nix-shell package update command, or something else?

Reply:
Also generated by `nix-shell -p niv --run "niv init -b nixpkgs-unstable"`.

You can run that in a `$(mktemp -d)` to confirm.